packages/fether-react/src/utils/withTokens.js

```diff
@@ -54,7 +72,8 @@ const withTokens = compose(
     return {
       tokensArray,
       tokensArrayWithoutEth: tokensArray.filter(
-        ({ address }) => address !== 'ETH' // Ethereum is the only token without address, has 'ETH' instead
+        // Ethereum and Ethereum Classic are the only tokens without address, has 'ETH' or 'ETC' instead
+        ({ address }) => address !== 'ETH' && address !== 'ETC'
```

Comment:
isNotErc20TokenAddress

Reply:
i'ved pushed in commit https://github.com/paritytech/fether/pull/483/commits/b94d4a28ca046dba4fa04b35820a9bfdfff07a69